[GEOS-10380] Features Templating - IncludeFlat directive for dynamic inclusion not properly working (#5644)
geoserver/geoserver: doc/en/user/source/community/features-templating/directives.rst

4) The ``subArray.json`` template (line 6) must be an array itself, the container array will be stripped and
   its values directly integrated inside ``anArray``.

> In case an includeFlat directive is specified and it's attribute value is a property interpolation directive, if the property name evaluates to a json it gets included flat in the final output e.g

Reviewer: ```suggestion
In case an includeFlat directive is specified, and it contains a property interpolation directive evaluating a json, it will get included flat in the final output e.g
```
